app/controllers/dashboards_controller.rb

```diff
@@ -0,0 +1,9 @@
+class DashboardsController < ApplicationController
+  def show
+    @environment = Environment.find(params[:id])
+    @data = Project.alphabetical.each_with_object({}) do |project, hash|
+      hash[project] = project.last_release_by_deploy_group
```

Comment:
I debated that, but in the end liked have the last_release_by_deploy_group environment agnostic. I'll change it if you feel strongly about it.